uaclient/tests/test_contract.py

```diff
@@ -384,26 +378,20 @@ def test_attached_config_refresh_machine_token_and_services(
                 }
             },
         }
+        new_token = copy.deepcopy(machine_token)
+        new_token["machineTokenInfo"]["contractInfo"]["resourceEntitlements"][
+            1
```

Comment:
Haha, thanks black!

Reply:
yes glad I don't have to worry about defending automated formatting. as this would have been a couple of cycles. I guess I could better construct locals to avoid that nastiness. But it's more fun to don't care

Reply:
Oh yeah, this is fine, it's totally worth the occasional dumb thing like this to not have to think about it at all the rest of the time.